README.md

```diff
@@ -39,28 +37,33 @@ possible.
 * ☐ `<material-menu>`
 * many more, including a fast table
 
-At this time we are not taking pull requests but please file an issue and we
-will work with you.
+At this time we are not taking pull requests, but please
+[file an issue](https://github.com/dart-lang/angular_components/issues)
+and we will work with you.
 
-**Officially Supported Browsers:** The last two versions of Chrome, Edge,
+**Officially supported browsers:** The last two versions of Chrome, Edge,
 Firefox, and Safari.
 
 ## Useful links
 
 * **[Example gallery](https://dart-lang.github.io/angular_components_example/)**
-* [AngularDart pub package](https://pub.dartlang.org/packages/angular2)
-* [AngularDart documentation](https://angulardart.org/)
-* [Material Design site](https://material.google.com/)
-* [Code Lab: AngularDart Components](https://webdev.dartlang.org/codelabs/angular2_components)
+* [AngularDart Components documentation](https://webdev.dartlang.org/components):
+  * [API reference](https://webdev.dartlang.org/components/api)
+  * [Codelab: AngularDart Components](https://webdev.dartlang.org/codelabs/angular2_components)
```

Comment:
I've made these changes.

@filiph the pubspec (in stagehand & in the codelab) says `angular2: ^3.0.0` and `angular_components: ^0.5.1`. Should we update to 3.1.0 and 0.5.2?